tests/tests_unit/test_data_classes/test_data_models/test_filters.py

```diff
@@ -185,8 +185,8 @@ def test_dump_filter(user_filter: Filter, expected: dict) -> None:
 
 
 def test_unknown_filter_type() -> None:
-    with pytest.raises(ValueError, match="Unknown filter type: unknown"):
-        Filter.load({"unknown": {}})
+    unknown = Filter.load({"unknown": {}})
+    assert isinstance(unknown, f.UnknownFilter)
```

Comment:
Maybe extend with a dump to ensure that loading and dumping a filter returns the same.
```suggestion
    assert isinstance(unknown, f.UnknownFilter)
    assert unknown.dump() == {"unknown": {}}
```